src/run.ts

```diff
@@ -206,11 +315,15 @@ async function destroy({
 // All valid arguments to dev should be enumerated here, this is the entrypoint to the script
 yargs(process.argv.slice(2))
   .command(
-    "local",
+    "watch",
```

Comment:
Oh neato, is this so that we can use cdk-watch and local separately to compare?

Reply:
originally I deleted the cdk watch stuff.  This is to not delete the cdk watch stuff.